src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -375,6 +375,9 @@ private static void userDisableHooks(Class<? extends Hook<?>> hookClass, boolean
 	public static final Option<Integer> variableChangesUntilSave = new Option<>("variable changes until save", 1000)
 		.setter(FlatFileStorage::setRequiredChangesForResave);
 
+	public static final Option<Boolean> simplifySyntaxesOnParse = new Option<>("simplify syntax on parse", true)
```

Comment:
i would include a comment that this option is intentionally omitted from the config. that is, it has to be added manually